util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java

```diff
@@ -36,18 +36,20 @@ public static String doTag(
 
 		return doTag(
 			_PAGE, formName, modelName, permissionPropagationCheckboxLabel,
-			false, pageContext);
+			false, false, pageContext);
```

Comment:
Based on the attribute's description
`<description>Whether the checkbox is selected by default. The default value is <![CDATA[<code>true</code>]]>.`
The default value should be true, right?

Reply:
Fixed it. Thanks